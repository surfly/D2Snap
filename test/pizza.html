version https://git-lfs.github.com/spec/v1
oid sha256:c21d4264427d990ca778a184eaa0bf3d3a588a65d08d4882f3e0dbc02e3c8648
size 603
