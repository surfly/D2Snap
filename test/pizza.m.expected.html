<section>
  # Our Pizza
  <div>
    ## Margherita
    A simple classic
    <button>Add</button>
    ## Capricciosa
    A rich taste
    <button>Add</button>
  </div>
</section>